Guesser/AbstractTypeGuesser.php

```diff
@@ -11,9 +11,9 @@
 
 namespace Sonata\PropelAdminBundle\Guesser;
 
+use Propel\Bundle\PropelBundle\Form\PropelTypeGuesser;
 use Sonata\AdminBundle\Guesser\TypeGuesserInterface;
 use Sonata\AdminBundle\Model\ModelManagerInterface;
-use Symfony\Bridge\Propel1\Form\PropelTypeGuesser;
```

Comment:
> I just got the feeling that our project is the only one which use this bundle since the BC break of february. 

Its not, [there are even packages that publicly depend on it, with last activity 6 month ago](https://packagist.org/packages/sonata-project/propel-admin-bundle/dependents)

But indeed [the stats are very very low](https://packagist.org/packages/sonata-project/propel-admin-bundle/stats) so let's not be too strict.

Good luck with the build!